Makefile

```diff
@@ -131,6 +140,10 @@ lint:
 	@echo "linting"
 	CGO_ENABLED=0 revive -formatter friendly -config revive.toml $$($(PACKAGES))
 
+swagger_spec:
+	go mod vendor
+	swag init --parseVendor -generalInfo server/api/router.go --exclude vendor/github.com/pingcap-incubator/tidb-dashboard --output docs/swagger
```

Comment:
Do we need to generate for `tidb-dashboard`? cc @HunDunDM

Reply:
and we need  to help users to install `swag` in scripts.

Reply:
> Thanks @loxp , Sorry too late for review. Do you test the swagger API? After build `pd-server` with `SWAGGER=1 make`, and swagger API not works.
> 
> ```
> ❯ curl http://127.0.0.1:2379/pd/api/v1/swagger
> Not Found
> ```
> 
> and also after I `curl` the API, the pd-server cannot be stoped by `Ctrl+C`.

I add a specific router for `/pd/api/v1/swagger` in the latest commit, PTAL.

PD is successfully stoped by `Ctrl+C` on my macOS Catalina. Is it different from your environment?

Reply:
> and we need to help users to install `swag` in scripts.

It is added in tools.go so can be installed in `make install-tools`.